.github/workflows/check-release.yml

```diff
@@ -1,27 +1,33 @@
 name: Check Release
+
 on:
   push:
     branches: ["*"]
   pull_request:
     branches: ["*"]
-  release:
-    types: [published]
   schedule:
     - cron: "0 0 * * *"
 
 jobs:
   check_release:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - uses: jupyterlab/maintainer-tools/.github/actions/base-setup@v1
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Setup environment
+        uses: jupyterlab/maintainer-tools/.github/actions/base-setup@v1
         with:
           python_version: "3.11.x"
+
       - name: Check Release
         uses: jupyter-server/jupyter_releaser/.github/actions/check-release@v2
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           version_spec: minor
+        env:
+          DEBUG: true  # Add this line to enable debugging output
```

Comment:
What additional debugging output is produced? Can you also include, in a comment, a link to any documentation which explains the `DEBUG` environment variable?